<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugLogger</class>
 <widget class="QWidget" name="DebugLogger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuplChecker Log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Vymazat</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="obnovit">
     <property name="text">
      <string>Obnovit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
